fluentd/Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM fluent/fluentd:stable-debian-onbuild
+FROM fluent/fluentd:edge-debian
 MAINTAINER Duan Shiqiang <duan@perxtech.com>
 
-RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev" \
+USER root
+
+RUN buildDeps="make gcc g++ libc-dev wget bzip2 gnupg dirmngr" \
   && apt-get update \
   && apt-get install -y --no-install-recommends $buildDeps \
-  && sudo gem install --no-document \
-        fluent-plugin-google-cloud \
+  && gem install --no-document \
+        #Do we ever use below plugin somewhere? 
```

Comment:
Currently we are not using this but eventually we need to support logging to Google Logging. So it's okay to disable this for now, we can enable it when they fix the issue with ruby 2.6 (already fixed in master branch)